eng/mgmt/automation/generation.yml

```diff
@@ -38,13 +46,27 @@ steps:
 - bash: |
     export PATH=$JAVA_HOME_11_X64/bin:$PATH
     java -version
+    set -x
+    ./eng/mgmt/automation/generate.py --tsp-config "$(TSP_CONFIG)" --version "$(VERSION)" --auto-commit-external-change --user-name "azure-sdk" --user-email "azuresdk@microsoft.com"
+  displayName: Generation from TypeSpec
+  condition: eq(variables.fromTypeSpec, true)
+
+- bash: |
+    export PATH=$JAVA_HOME_11_X64/bin:$PATH
+    java -version
+    set -x
     ./eng/mgmt/automation/generate.py --readme "$(README)" --tag "$(TAG)" --autorest-options="$(AUTOREST_OPTIONS)" --service "$(SERVICE)" --version "$(VERSION)" --suffix "$(SUFFIX)" --auto-commit-external-change --user-name "azure-sdk" --user-email "azuresdk@microsoft.com"
-  displayName: Generation
+  displayName: Generation from Swagger
+  condition: eq(variables.fromTypeSpec, false)
 
 - template: /eng/common/pipelines/templates/steps/create-pull-request.yml
   parameters:
     PRBranchName: fluent-lite-generation-$(Build.BuildId)
-    CommitMsg: '[Automation] Generate Fluent Lite from $(README)#$(TAG)'
-    PRTitle: '[Automation] Generate Fluent Lite from $(README)#$(TAG)'
+    ${{ if variables.fromTypeSpec }}:
+      CommitMsg: '[Automation] Generate Fluent Lite from TypeSpec $(README)'
+      PRTitle: '[Automation] Generate Fluent Lite from TypeSpec $(README)'
+    ${{ else }}:
+      CommitMsg: '[Automation] Generate Fluent Lite from $(README)#$(TAG)'
+      PRTitle: '[Automation] Generate Fluent Lite from $(README)#$(TAG)'
```

Comment:
Should we have these use Swagger to make it more obvious where things are still using it instead of TypeSpec?

Reply:
Yeah, we could. Added.